### DISCUSSION TITLE
auth: how to detect new user (like onCreate event)

### DISCUSSION THREAD

**Discussion by Aure77** [invertase/react-native-firebase#4416] [Q&A] [answered]
I would like to achieve that (`functions.auth.user().onCreate(...)`):
https://firebase.google.com/docs/functions/auth-events

but @react-native-firebase/auth don't seems to have a listener that allow to detect a new user (_onCreate_ or _onDelete_).

Is it possible with @react-native-firebase/auth ?

**Comment by mikehardy** [comment, marked as answer]
If it's not documented in the https://rnfirebase.io/reference/auth area, and/or does not exist on the underlying SDKs

- https://firebase.google.com/docs/reference/android/com/google/firebase/auth/package-summary
- https://firebase.google.com/docs/reference/ios/firebaseauth/api/reference/Classes

...then no, and I don't think it does? There are a few cases where the "Admin SDK" (which you have access to in the functions environment on the cloud servers) has methods not available to the client libraries and this feels like it would be one of them

**Reply by andersonaddo** [reply]
Iirc, this is an admin sdk-only feature. You're going to have to make use of the cloud trigger and find a way for it to pass down the information you want to the clients in question.
